### PR TITLE
Feature/load screens

### PR DESCRIPTION
**Pull request by 10dimensions**
- Setup Load Effect before all needed data is retrieved
- Setup Prompts (Screens) for missing Web, Gallery content

To-Do:
- Prompt Screen for Un-purchased Parcel

Note:
Not to be merged, but reviewed, as data is forcefully made `null`

### CODE REVIEW

**Comment by codynhat** [comment]
I see the error you were mentioning in the preview deployment. This may be due to breaking changes between Ceramic streams. Where is it currently loading the stream ID from? A hardcoded land parcel?

You should be unblocked by Ceramic now that they have released 1.0. Try updating `@ceramicnetwork/http-client` to `1.0.0`. If there are still errors, we may need to wait to create new documents in the Cadastre.

**Comment by 10dimensions** [comment]
Setup Load Effect before all needed data is retrieved
Setup Prompts (Screens) for missing Web, Gallery content
Prompt Screen for Un-purchased Parcel
GPS reverted to sensor co-ordinates

Note:
Can be merged with `main`
Separate branch `demo` setup for showcase purposes

**Comment by 10dimensions** [comment]
- Handle Conflict while merge
- Handle Exception while Querying Ceramic StreamId
- Fix Background Image Height to Scroll-Y

File: src/api/api.js [hunk 1]
```diff
@@ -83,21 +83,14 @@ const getParcelInfo = async(id) => {
...
-    name: "Dal Lake",
-    webContent: "https://www.beautifulworld.com/asia/india/dal-lake/",
-    mediaContent: [
-      {"name": "Lake Boat", "contentUrl" : "QmSJZ2DvrLz2hBeduYiPTsSmCXViZrbj1cc2V3mud8kX6N"},
-      {"name": "Mountain Bike", "contentUrl" : "bafybeifvwcgfg6he3ihovvhakrdlihh5ol4rucu26ou5zel3jlhgo4acay"}
-    ]
+  let doc = null;
+  
+  try{
+    doc = await ceramic.loadStream(docid);
```
Comment (codynhat) [inline src/api/api.js:89]: See the new note here in Ceramic docs: 
- https://developers.ceramic.network/build/queries/#query-a-stream
- https://developers.ceramic.network/streamtypes/tile-document/api/#load-a-document

There is a new API specifically for Tile documents `TileDocument.load`. I'm not sure if you need to use it or not. It seems like it is only needed for making changes?

**Review by codynhat — APPROVED** [review]

**Pull request merged**